docker/e2e-service-consumer/Dockerfile

```diff
@@ -0,0 +1,5 @@
+FROM apache/skywalking-java-agent:8.9.0-java8
```

Comment:
I have concerned about this. e2e of main repo has assigned the commit id of agent.

Could you check it? I think we have this mechanism already.

Reply:
Yes. I've noticed this part. But it exists in the `env` file which is read by the `infra-e2e` toolkit.

Shall we introduce a build argument as a `TAG`?